raysort/sort_utils.py

```diff
@@ -307,8 +341,13 @@ def validate_output(cfg: AppConfig):
     total = sum(sz for sz, _ in results)
     assert total == cfg.total_data_size, total - cfg.total_data_size
     all_checksum = b"".join(chksm for _, chksm in results)
+    with open(get_manifest_file(cfg), "r") as fin:
+        reader = csv.reader(fin)
+        input_checksums = [int(row[-1], 16) for row in reader]
+
     with tempfile.NamedTemporaryFile() as fout:
         fout.write(all_checksum)
         fout.flush()
-        _run_valsort(f"-s {fout.name}")
+        output_summary = _run_valsort(f"-s {fout.name}")
+        compare_checksums(input_checksums, str(output_summary))
     logging.info("All OK!")
```

Comment:
Won't this still print `All OK!` even if the checksums don't match? Probably a nit, but may be good to make sure it doesn't print that.

Reply:
Yep. Making `compare_checksums` return a bool should fix this.